xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/commentsinline.vm

```diff
@@ -229,7 +229,9 @@ $xwiki.ssfx.use('uicomponents/viewers/comments.css', true)
         <input type="hidden" name="${xCommentClass}_replyto" value="$!replyTo"/>
         <div class="commentcontainer">
           <div class="commenteditor">
-              <!-- loaded by javascript (see comments.js) -->
+            <!-- loaded by javascript (see comments.js) -->
+            <!-- stores the submitted comment in case of error during the submission (eg, invalid captcha). -->
+            <div class="hidden" id="submittedcomment">$!comment</div>
```

Comment:
any reason to not use an input type hidden here?

Reply:
If it is stored in an hidden field, it will needlessly be submitted on the next submission of the form.

Reply:
And can't you control when to output this div, something like: 
```
#if ($request.error)
  <input type="hidden" id="submittedcomment" value="$!comment" />
#end 
```

Reply:
Your are right, I'm now using `$captchaAnswerWrong` as the condition to add the tag.